src/BenchmarkDotNet.Core/Exporters/Xml/SimpleXmlWriter.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+using System.IO;
+using System.Xml;
+
+namespace BenchmarkDotNet.Exporters.Xml
+{
+    public class SimpleXmlWriter : IXmlWriter, IDisposable
```

Comment:
I suggest that only `XmlExporter` should be public. Our xml serializers, writers and corresponding interfaces should be internal. Our users don't need to know about their existence and the variety of choice you give them with `Full` and `Simple` version should prevent anybody from deriving a custom class.